main.tf

```diff
@@ -300,10 +259,13 @@ module "eks" {
   workers_role_name                              = var.workers_iam_role_name
   manage_cluster_iam_resources                   = var.cluster_iam_role_name == null ? true : false
   cluster_iam_role_name                          = var.cluster_iam_role_name
+  worker_create_security_group                   = false
+  worker_security_group_id                       = local.workers_security_group_id
+  cluster_create_security_group                  = false
+  cluster_security_group_id                      = local.cluster_security_group_id
 
   workers_group_defaults = {
     tags                                 = var.autoscaling_enabled ? [ { key = "k8s.io/cluster-autoscaler/${local.cluster_name}", value = "owned", propagate_at_launch = true }, { key = "k8s.io/cluster-autoscaler/enabled", value = "true", propagate_at_launch = true} ] : null
-    additional_security_group_ids        = [local.security_group_id]
```

Comment:
Not sure why this was deleted. More context needed.

Reply:
since we are bringing our own workers security group we already have all the security rules we need (as described here: https://github.com/sassoftware/viya4-iac-aws/blob/byo_sg/docs/user/BYOnetwork.md#workers-security-group)
So there was no need to tell the cluster module to tack on our main security group (which was overkill anyway - the new way is more precise because it ONLY adds what is needed). Another reason why always creating our own seemed cleaner.

Reply:
Again, if BYO SGs are not provided what's the fall back and how does this change our current workflow.